README.md

```diff
@@ -161,3 +161,37 @@ The list of developer tools is:
 
 [secrets-manager]: https://bitwarden.com/products/secrets-manager/
 [bws-help]: https://bitwarden.com/help/secrets-manager-cli/
+
+## Cargo fmt
+
+In order to sort and group imports, as well as wrap long comments, we need the nightly version of
+cargo-fmt.
```

Comment:
Do we need to mention why we use it. I think it's sufficient to just say `We use nightly version of `cargo-fmt`.

Reply:
Mostly I wanted to add this for new developers looking into the SDK. Maybe this would be better for the contributing docs, and just leave the `We use nightly version of cargo-fmt` note here?

Reply:
You can use a generic phrasing. Something like `We use certain unstable features of cargo fmt which requires the nightly version of cargo-fmt`.

Reply:
Updated in [31b7733](https://github.com/bitwarden/sdk/pull/793/commits/31b773334d0fb7fbf437b051f0a75ae300d96e0c)